pygmt/tests/test_datasets_earth_free_air_anomaly.py

```diff
@@ -0,0 +1,82 @@
+"""
+Test basic functionality for loading Earth free air anomaly datasets.
+"""
+import numpy as np
+import numpy.testing as npt
+import pytest
+from pygmt.datasets import load_earth_free_air_anomaly
+from pygmt.exceptions import GMTInvalidInput
+
+
+def test_earth_faa_fails():
+    """
+    Make sure earth_free_air_anomaly fails for invalid resolutions.
+    """
+    resolutions = "1m 1d bla 60d 001m 03".split()
+    resolutions.append(60)
+    for resolution in resolutions:
+        with pytest.raises(GMTInvalidInput):
+            load_earth_free_air_anomaly(resolution=resolution)
+
+
+def test_earth_faa_incorrect_registration():
+    """
+    Test loading earth_free_air_anomaly with incorrect registration type.
+    """
+    with pytest.raises(GMTInvalidInput):
+        load_earth_free_air_anomaly(registration="improper_type")
+
```

Comment:
These tests are common to all Earth datasets. Perhaps we can move all these tests in a new test file like `test_datasets_load_remote_dataset.py` instead.

Reply:
This can be done in a separate PR if necessary.